tools/codegen/model.py

```diff
@@ -605,6 +605,11 @@ class BackendIndex:
     # Other backend-specific information that is on a per-operator basis
     index: Dict['OperatorName', BackendMetadata]
 
+    # Logging macros that are inserted into wrappers. Only really used by external backends.
+    per_op_log: Optional[str] = None
+    per_argument_log: Optional[str] = None
+    cpu_fallback_counter: Optional[str] = None
```

Comment:
some tests exercising this?

Reply:
welp, I added some expect tests but forgot to commit them. Adding them back soon.